Explorer/Assets/DCL/SDKComponents/RealmInfo/WriteRealmInfoSystem.cs

```diff
@@ -0,0 +1,57 @@
+using Arch.Core;
+using Arch.SystemGroups;
+using CrdtEcsBridge.Components;
+using CrdtEcsBridge.ECSToCRDTWriter;
+using DCL.ECSComponents;
+using DCL.Multiplayer.Connections.RoomHubs;
+using DCL.Utilities;
+using ECS;
+using ECS.Abstract;
+using ECS.Groups;
+
+namespace DCL.SDKComponents.RealmInfo
+{
+    [UpdateInGroup(typeof(SyncedInitializationSystemGroup))]
+    public partial class WriteRealmInfoSystem : BaseUnityLoopSystem
+    {
+        private readonly IECSToCRDTWriter ecsToCRDTWriter;
+        private readonly ObjectProxy<IRealmData> realmDataProxy;
+        private readonly ObjectProxy<IRoomHub> roomHubProxy;
+
+        internal WriteRealmInfoSystem(World world, IECSToCRDTWriter ecsToCRDTWriter, ObjectProxy<IRealmData> realmDataProxy, ObjectProxy<IRoomHub> roomHubProxy) : base(world)
+        {
+            this.ecsToCRDTWriter = ecsToCRDTWriter;
+            this.realmDataProxy = realmDataProxy;
+            this.roomHubProxy = roomHubProxy;
+        }
+
+        protected override void Update(float t)
+        {
+            if (!realmDataProxy.Configured
+                || !roomHubProxy.Configured
+                || !realmDataProxy.Object!.IsDirty
+                || !realmDataProxy.Object.Configured)
+                return;
+
+            PropagateToScene();
+
+            realmDataProxy.Object.IsDirty = false;
```

Comment:
You can't reset `dirty` from the scene system as the proxy and `IRealmData` is shared and it will be reset on the first executed scene and other scenes will not propagate any data.

Please check the lifecycle of `IRealmData` but I suppose it's impossible to change it while the scene in running: scenes will die first and then realm will be changed. `IsDirty` and `Configured` are only suitable for the global world as it's running all the time

Reply:
Ok I guess I'll have to create a specific plugin and system in the global world only to reset it then

Reply:
at https://github.com/decentraland/unity-explorer/pull/838/commits/62fc44d7e74b697600b976ca9780526b53457f90 I created a mini plugin and system to reset the `RealmData.IsDirty` flag during cleanup phase.

Something to take into account:
* Realistically whenever the realm is changed, the scenes will be unloaded and new scenes will be loaded (the `RealmInfo` component will be added in those new scenes at Initialization)
* The `IsDirty` flag would only be relevant when we are dealing with "global scenes" like Portable Experiences, that are still not supported in E@ I think.

Reply:
yeah, cool, it totally makes sense